nav2_robot/include/nav2_robot/robot.hpp

```diff
@@ -31,8 +31,11 @@ class Robot
   explicit Robot(rclcpp::Node::SharedPtr & node);
   Robot() = delete;
 
-  bool getCurrentPose(geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr & robot_pose);
-  bool getCurrentVelocity(nav_msgs::msg::Odometry::SharedPtr & robot_velocity);
+  bool getGlobalLocalizerPose(
+    geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr & robot_pose);
```

Comment:
This is such a long typename. Adding the line `using geometry_msgs::msg::PoseWithCovarianceStamped` in the class would let you shorten these names here and in the CPP file.

Reply:
I won't be using the `using` in this PR to be consistent. However, you are correct we should consider `using`.